Makefile

```diff
@@ -34,10 +34,8 @@ NGXSYMCHECK	= tools/ngx_symcheck
 #
```

Comment:
TAPE and NPM on the lines above should be vestigial now.

Reply:
Nice catch!